central/delegatedregistryconfig/delegator/delegator_test.go

```diff
@@ -242,6 +243,10 @@ func TestGetDelegateClusterID(t *testing.T) {
 }
 
 func TestDelegateEnrichImage(t *testing.T) {
+	if features.FlattenImageData.Enabled() {
+		t.Skip("ROX_FLATTEN_IMAGE_DATA flag is enabled")
+	}
```

Comment:
nit: can continue testing this by changing the env var as part of the test.

```suggestion
	testutils.MustUpdateFeature(t, features.FlattenImageData, false)
```